problems/2632_сurry/curry.test.js

```diff
@@ -0,0 +1,31 @@
+const {curry} = require('./curry');
+
+describe('sum of arguments', () => {
+   it.each([
+       [[1],[2],[3]],
+       [[1,2],[3]],
+       [[],[1],[],[2],[3]]
+   ])("function should return the same value as sum(1, 2, 3)", (...args) => {
+      const curriedSum = curry((a, b, c) => a + b + c);
+      const sum = [...args].reduce((res,el) => res = curriedSum(...el), undefined);
```

Comment:
```suggestion
      const sum = args.reduce((_, el) => curriedSum(...el), undefined);
```

1. `res =` – в данном случае излишен, поэтому сразу возвращаем значение

     если оставлять варик с `res =`, то читаемей будет так
    ```ts
      const sum = args.reduce((res, el) => {
        res = curriedSum(...el);
        return res;
      }, undefined);
    ```
2. `[...args].reduce`  – в синтаксисе `function someFn(...args) {}` получается так, что `args`это уже функция, поэтому можно не создавать новую через `[...args]`.